core/vm/interface.go

```diff
@@ -24,18 +26,29 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
+type AccountExtraDataStateReader interface {
+	// Return nil for public contract
+	ReadPrivacyMetadata(addr common.Address) (*state.PrivacyMetadata, error)
+	ReadManagedParties(addr common.Address) ([]string, error)
+}
+
+type AccountExtraDataStateWriter interface {
+	WritePrivacyMetadata(addr common.Address, pm *state.PrivacyMetadata)
```

Comment:
MINOR: For consistency with the MinimalApiState I would rename those function `Set...` better than `Write...`

Reply:
changed